tests/unit_tests/base_test.py

```diff
@@ -62,7 +57,16 @@ def request_colorized_icon(
         origin=ORIGIN_FOR_TESTING
     ):
         return self.app.get(
-            build_request_url_for_icon(icon_name, scale, red, green, blue, icon_category),
+            url_for(
+                'colorized_icon',
+                icon_set_name=icon_category,
+                icon_name=icon_name,
+                scale=scale,
+                red=red,
+                green=red,
+                blue=blue,
+                _external=True
```

Comment:
`_external=True` should not be needed and use in unittest.

Reply:
All `_external=True` are now deleted, except 3 places where it is necessary.